### PR TITLE
GitLab: Enable prometheus scrapes from k8s

### PR DESCRIPTION
**Pull request by leafty**
Note: admin still has to follow https://gitlab.com/help/administration/monitoring/prometheus/gitlab_metrics#gitlab-prometheus-metrics

After these steps, I could do:
```bash
$ kubectl -n renku exec -it renku-gitlab-ffc56b8f7-98q7s curl -- "http://localhost/gitlab/-/metrics"
# HELP gitlab_banzai_cacheless_render_real_duration_seconds Multiprocess metric
# TYPE gitlab_banzai_cacheless_render_real_duration_seconds histogram
gitlab_banzai_cacheless_render_real_duration_seconds_bucket{controller="HelpController",action="index",le="+Inf"} 30
gitlab_banzai_cacheless_render_real_duration_seconds_bucket{controller="HelpController",action="index",le="0.001"} 0
gitlab_banzai_cacheless_render_real_duration_seconds_bucket{controller="HelpController",action="index",le="0.01"} 0
gitlab_banzai_cacheless_render_real_duration_seconds_bucket{controller="HelpController",action="index",le="0.1"} 22
gitlab_banzai_cacheless_render_real_duration_seconds_bucket{controller="HelpController",action="index",le="1"} 30
gitlab_banzai_cacheless_render_real_duration_seconds_count{controller="HelpController",action="index"} 30
gitlab_banzai_cacheless_render_real_duration_seconds_sum{controller="HelpController",action="index"} 2.8562915325164795
[...]
```

### CODE REVIEW

**Comment by rokroskar** [comment]
This won't be enough, right? I thought the metrics directory has to also be defined.

**Comment by leafty** [comment]
yeah, it's not working as is.

File: charts/gitlab/Chart.yaml [hunk 1]
```diff
@@ -2,4 +2,4 @@ apiVersion: v1
 appVersion: '1.0'
 description: A Helm chart for the Renku Gitlab server
 name: gitlab
-version: 0.1.0
+version: 0.0-unclean-4b34c5f
```
Comment (rokroskar) [inline charts/gitlab/Chart.yaml]: chartpress fail

File: charts/renku/Chart.yaml [hunk 2]
```diff
@@ -2,4 +2,4 @@ apiVersion: v1
 appVersion: '1.0'
 description: A Helm chart for the Renku platform
 name: renku
-version: 0.1.0
+version: 0.0-unclean-4b34c5f
```
Comment (rokroskar) [inline charts/renku/Chart.yaml]: chartpress fail

File: charts/renku/values.yaml [hunk 3]
```diff
@@ -552,7 +552,7 @@ jupyterhub:
   hub:
     image:
       name: renku/jupyterhub-k8s
-      tag: latest
+      tag: 0.0-unclean-4b34c5f
```
Comment (rokroskar) [inline charts/renku/values.yaml]: chartpress fail

File: charts/renku/values.yaml [hunk 4]
```diff
@@ -608,7 +608,7 @@ jupyterhub:
   singleuser:
     image:
       name: renku/singleuser
-      tag: latest
+      tag: 0.0-unclean-4b34c5f
```
Comment (rokroskar) [inline charts/renku/values.yaml]: chartpress fail

File: charts/renku/values.yaml [hunk 5]
```diff
@@ -637,12 +637,12 @@ jupyterhub:
 apispec:
   image:
     repository: renku/apispec
-    tag: latest
+    tag: 0.0-unclean-4b34c5f
```
Comment (rokroskar) [inline charts/renku/values.yaml]: chartpress fail

File: charts/renku/values.yaml [hunk 6]
```diff
@@ -637,12 +637,12 @@ jupyterhub:
...
 
 tests:
   image:
     repository: renku/tests
-    tag: latest
+    tag: 0.0-unclean-4b34c5f
```
Comment (rokroskar) [inline charts/renku/values.yaml]: chartpress fail

**Comment by rokroskar** [comment]
so confused... in one instant all the tags were wrong, but now they seem fine

**Pull request merged**